docs/2022-07-18-JSON-RPC-Server-API.md

```diff
@@ -292,9 +293,19 @@ where `original-term`, `original-term-index`, `substitution` and `rewritten-term
   }
 }
 ```
-
 where `rewritten-term` is optional and `origin` is one of `kore-rpc`, `booster` or `llvm`.
 
+When `log-timing` is set to `true`, the response to `execute`, `simplify` and `implies` requests will contain information about the wall-clock time spent in different components (`kore-rpc`, `booster`, `proxy`) or overall (without qualifiing tag) while processing the request.
+
+```json
+{
+  "tag": "processing-time",
+  "timing": [[null, 69.12], ["kore-rpc", 12.34], ["booster", 56.78]]
```

Comment:
The documentation needs to be changed to reflect the current definition of the `ProcessingTime` log entry.

Reply:
Yes, forgot that, thanks for reminding me!